### ISSUE TITLE
mean_psnr always been 0 when I was trainning carn model

### ISSUE THREAD

**Issue by paullu** [nmhkahn/CARN-pytorch#4]
@nmhkahn thank you for your codes. When I was trainning a carn mode as you described in README, mean_psnr always been 0 。Does it reasonable？

**Comment by nmhkahn** [comment]
Hi.
When I updated the repo lately, I set default validation set as Urban100 for debugging purpose.
So, I guess you don't have Urban100 data in the default directory (`dataset/Urban100`).
